app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
```

Comment:
- [ ] Great job so far, however. You are missing  a main static route Kindly add create one and make sure:
1. the created route is marked as `root`.
2. A controller with one action was created
3. A view for the root route is created (`root#index`)
   * It should include only `<div id="root">` The rest should be done with React and Redux.

Reply:
Everything is well set up thanks